cypress/utils/dashboards/data_explorer/commands.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DATA_EXPLORER_PAGE_ELEMENTS } from './elements.js';
+
+/**
+ * Get the New Search button.
+ */
+Cypress.Commands.add('getNewSearchButton', () => {
+  return cy.getElementByTestId(DATA_EXPLORER_PAGE_ELEMENTS.NEW_SEARCH_BUTTON, { timeout: 10000 });
+});
```

Comment:
The current consensus is that all non data explorer page specific functionality (search bar etc.) should be a global command. Open to reopening this discussion as I agree it can have negative future repercussions.

This button refers to the "new search" functionality, not a search button that is new.
<img width="347" alt="image" src="https://github.com/user-attachments/assets/3deb48aa-0085-46c7-b68f-310a190330b4" />